### PR TITLE
devenv writes to different log files for ebrelayer, witness, sifnoded

### PR DESCRIPTION
**Pull request by banshee**

### CODE REVIEW

File: cmd/ebrelayer/main.go [hunk 1]
```diff
@@ -211,6 +210,7 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 
 	logConfig := zap.NewDevelopmentConfig()
 	logConfig.Sampling = nil
+	logConfig.Encoding = "json"
```
Comment (Brando753) [inline cmd/ebrelayer/main.go:213]: Are you hardcoding log output as Json and not using a flag, is that something we always want to be JSON?

File: cmd/ebrelayer/main.go [hunk 2]
```diff
@@ -337,6 +336,7 @@ func RunInitWitnessCmd(cmd *cobra.Command, args []string) error {
 	validatorMoniker := args[4]
 
 	logConfig := zap.NewDevelopmentConfig()
+	logConfig.Encoding = "json"
```
Comment (Brando753) [inline cmd/ebrelayer/main.go:339]: Same question as above

File: smart-contracts/src/devenv/ebrelayer.ts [hunk 3]
```diff
@@ -114,7 +120,8 @@ export class RelayerRunner extends ShellCommand<EbrelayerResults> {
     readonly chainNet = "localnet",
     readonly ebrelayerDB = `levelDB.db`,
     readonly relayerdbPath = "./relayerdb",
-    readonly symbolTranslatorFile = "../test/integration/config/symbol_translator.json"
+    readonly symbolTranslatorFile = "../test/integration/config/symbol_translator.json",
+    readonly logFile = "/tmp/sifnode/ebrelayer.log"
```
Comment (Brando753) [inline smart-contracts/src/devenv/ebrelayer.ts:124]: Lets change this to relayer.log as ebrelayer is the same codebase for the relayers and witnesses. Also while this is fine for now we will have to think about how we want to sanely handle default file names for multiple instances of the relayers/witnesses so relayer-1.log relayer-2.log from the readonly constructor.

**Review by Brando753 — CHANGES_REQUESTED** [review]

File: cmd/ebrelayer/main.go (hunk 1 above)
Comment (banshee) [reply cmd/ebrelayer/main.go:213]: Yes, right now it's hardcoded.  We should probably add a feature to let users select this, but I thought we should get it working with json first.

File: smart-contracts/src/devenv/ebrelayer.ts (hunk 3 above)
Comment (banshee) [reply smart-contracts/src/devenv/ebrelayer.ts:124]: Let's talk; I think we need a new word

**Comment by Brando753** [comment]
Tested on docker, it works.

**Review by Brando753 — APPROVED** [review]

**Pull request merged**